src/localstorage.js

```diff
@@ -0,0 +1,36 @@
+function LS() {}
+LS.prototype.fetchItem = function () {
+  let items = localStorage.getItem('items');
+  if (items) {
+    items = JSON.parse(items);
+  } else {
+    items = [];
+  }
+  return items;
```

Comment:
- [ ] I think you need to take a close look at this record, the first issue is, the user is able to insert empty todo and the second one is, that anytime the user refreshes the app, all the data is lost. please work on fixing it

![6](https://github.com/Obote/todo-list/assets/77843050/9dba3bee-0b8e-46a6-b179-1e60199eeb69)